Update README (#2802)
ansible-collections/community.general: README.md

[![Build Status](https://dev.azure.com/ansible/community.general/_apis/build/status/CI?branchName=main)](https://dev.azure.com/ansible/community.general/_build?definitionId=31)
[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/community.general)](https://codecov.io/gh/ansible-collections/community.general)

> This repo contains the `community.general` Ansible Collection. The collection is a part of Ansible package and includes many modules and plugins supported by Ansible community which are not part of more specialized community collections.

Reviewer: ```suggestion
This repository contains the `community.general` Ansible Collection. The collection is a part of the Ansible package and includes many modules and plugins supported by Ansible community which are not part of more specialized community collections.
```

- name: community.general
```

> Note that if you install the collection manually, it will not be upgraded automatically with upgrade of the Ansible package. To upgrade the collection to the latest available version, run the following command:

Reviewer 2: I'm not sure about this.
```suggestion
Note that if you install the collection manually, it will not be upgraded automatically with upgrading of the Ansible package. To upgrade the collection to the latest available version, run the following command:
```

Reviewer 3: Maybe this will make it clearer: `it will not be upgraded automatically when you upgrade the Ansible package.`

Reviewer: I was about to suggest something similar, but it had more passive voice, so this one is definitely better :)

Author: Committed, thanks!

```bash
ansible-galaxy collection install community.general --upgrade
```

You can also install a specific version of the collection, for example, if you need to downgrade when something is broken in the latest version (please report an issue in this repository). Use the following syntax where `X.Y.Z` can be any [available version](https://galaxy.ansible.com/community/general):

```bash
ansible-galaxy collection install community.general:==X.Y.Z
```

See [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.

## Contributing to this collection

The content of this collection is made by good people like you, a community of individuals collaborating on making the world better through developing automation software.

All types of contributions are very welcome.

> You don't know how to start? Refer to our [contribution guide](CONTRIBUTING.md)!

Reviewer: Otherwise the link won't work from galaxy.
```suggestion
You don't know how to start? Refer to our [contribution guide](https://github.com/ansible-collections/community.general/blob/main/CONTRIBUTING.md)!
```

Author: Good spot, thanks!

> The current maintainers are listed in the [commit-rights.md](https://github.com/ansible-collections/community.general/blob/main/commit-rights.md#people) file. Don't hesitate to reach them out mentioning in the proposals.

Reviewer 3: "Don't hesitate to reach out to them by mentioning them in the proposals."

Author: @cybette how about the new wording? PTAL

### Communication

> We announce releases and important changes through the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546). Be sure you are subscribed.

Reviewer 2: I'm not sure if this is necessary here, as that issue's notification, maybe too much for a user wanting to be notified about `c.g.` only.

Author: There are also info about core releases, other collections, and contributor summits. I'll add a note that "If you are a contributor blah blah"

Author: done, PTAL

> We announce important development changes and releases through the [Ansible Bullhorn newsletter](http://eepurl.com/gZmiEP). If you are a collection developer, be sure you are subscribed.

Reviewer 2: ```suggestion
We announce important development changes and releases through the [Ansible Bullhorn newsletter](https://eepurl.com/gZmiEP). If you are a collection developer, be sure you are subscribed.
```

Author: Fixed, thanks!

> Join us in the ``#ansible-community``and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels).

Reviewer: Hmm, shouldn't we also mention #ansible for support? Otherwise we might get more visitors asking for support that need to be redirected.

Author: Sounds sensible, added, thanks

> Join us in the `#ansible` (general use questions and support), `#ansible-community` (community and collection development questions), and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels).

Reviewer: ```suggestion
Join us in the `#ansible` (general use questions and support), `#ansible-community` (community and collection development questions), and other [IRC channels](https://docs.ansible.com/ansible/devel/community/communication.html#irc-channels) on [Libera.chat](https://libera.chat).
```
Mentioning the IRC network is important :)

> We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.

Reviewer 2: What do you think about actually subscribing to Bullhorn newsletter instead of watching the issue? From top post in issue they can be subscribed here (http://eepurl.com/gZmiEP).

@cybette Is this the official method?

Reviewer 3: Yes, subscribing with [this link](http://eepurl.com/gZmiEP) is the official way. Alternatively you can link to the [News wiki](https://github.com/ansible/community/wiki/News#the-bullhorn). I do not recommend linking to the [issue](https://github.com/ansible/community/issues/546) because as @aminvakil said it can be too noisy for most users, plus we will probably create a new issue soon as the old one is getting quite long.

Author: Sounds sensible, changed, thanks!

> We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.

Author: ```suggestion
We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](http://eepurl.com/gZmiEP) and join us.
```

> We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.

Reviewer 2: ```suggestion
We take part in the global quarterly [Ansible Contributor Summit](https://github.com/ansible/community/wiki/Contributor-Summit) virtually or in-person. Track the [Ansible Bullhorn newsletter](https://github.com/ansible/community/issues/546) and join us.
```

Reviewer: or `Track Ansible's The Bullhorn newsletter ...`? Might be clearer, since the newsletter isn't called "Ansible Bullhorn", but "The Bullhorn".

Author: I changed the first mentioning of The Bullhorn to `Ansibles [The Bullhorn newsletter]..` and left this sentence almost as it was. Not to repeat `Ansible's`. Thanks!

Reviewer 2: Aha, that was the reason I made this suggestion :)
I thought they should be identical. Feel free to ignore or change it to something else.

Author: thanks @felixfontein for clarifying:)
